lib/compat/wordpress-trunk/index.php

```diff
@@ -0,0 +1,8 @@
+<?php
+/**
+ * Temporary compatibility shims for features present in Gutenberg.
```

Comment:
I can't specify version such as 5.8.1 because I can't be certain when this change gets backported. I guess this means that on every core release we'll have to move all the PHP code `compat/wordpress-trunk` to `compat/wordpress-$CURRENT_VERSION`